transformations/syntactically_diverse_paraphrase/transformation.py

```diff
@@ -0,0 +1,55 @@
+import benepar
+import spacy
+import nltk
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+from transformations.syntactically_diverse_paraphrase.sowreap.parse_utils import (
+    Sentence,
+)
+from transformations.syntactically_diverse_paraphrase.sowreap.reap_utils import (
+    reapModel,
+)
+from transformations.syntactically_diverse_paraphrase.sowreap.sow_utils import (
+    sowModel,
+)
+
+
+class ParaphraseSowReap(SentenceOperation):
+    tasks = [TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
```

Comment:
You need to add a [`keywords` attribute](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md)

Reply:
Done